src/cdk-experimental/menu/menu-interface.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {InjectionToken} from '@angular/core';
+
+/** Injection token used to return classes implementing the Menu interface */
+export const CDK_MENU = new InjectionToken<Menu>('cdk-menu');
+
+/** Interface which specifies Menu operations and used to break circular dependency issues */
+export interface Menu {
```

Comment:
We should make sure that this symbol isn't exported through the public API when we get that far